localization/english/religion/wc_religion_l_english.yml

```
@@ -277,6 +277,12 @@
  cult_of_torga_adherent_plural:0 "Torgi"
  cult_of_torga_desc:0 "A pantheon of Loa founded by the tortollans. Torga, a turtle Loa, is the main Loa of this pantheon. He is a wise and benevolent Loa, and his followers frequently undertake pilgrimages to him to hear his stories."
  
+ faithless:0 "Kortheki"
+ faithless_adj:0 "Kortheki"
+ faithless_adherent:0 "Kortheki"
+ faithless_adherent_plural:0 "Kortheki"
+ faithless_desc:0 "Also called the Faithless, the Kortheki are those who turned their away from their serpent loa, Sethraliss. Heretics, they gave up the peaceful path to become machines of war, using the divine essence by force to fulfill their interests and their destiny."
```

Suggested some changes.
```suggestion
 faithless_desc:0 "Also called the Faithless, the Kortheki are those who turned away from their serpent loa, Sethraliss. The heretics turned away from the path of peace to become machines of war, forcefully using the divine essence of Sethraliss to fulfill their interests and their destiny."
```

---

Maybe "turned their back on their serpent loa" in the first sentence?

---

Works for me, I had been torn between the two. Updated accordingly.

---

I choose the first proposition